src/universal/modules/meeting/components/SummaryFirstTime/SummaryFirstTime.js

```diff
@@ -6,6 +6,7 @@ import ui from 'universal/styles/ui';
 import {Type} from 'universal/components';
 import CopyShortLink from 'universal/modules/meeting/components/CopyShortLink/CopyShortLink';
 import FontAwesome from 'react-fontawesome';
+import emoji100 from 'universal/styles/theme/images/emoji/apple-emoji-100@3x.png';
```

Comment:
How about putting things in static for now? Then, we can shift that whole folder to S3 as part of our build process...